SharpLink/LavalinkPlayer.cs

```diff
@@ -205,7 +195,7 @@ internal async Task UpdateSessionAsync(SessionChange change, object changeData =
                         JObject data = new JObject();
                         data.Add("op", "voiceUpdate");
                         data.Add("guildId", voiceServer.Guild.Id.ToString());
-                        data.Add("sessionId", sessionId);
+                        data.Add("sessionId", (await VoiceChannel.Guild.GetCurrentUserAsync()).VoiceSessionId);
```

Comment:
You mentione that your intention was to also make this hopefully work for reconnect problems where the bot is already in a channel. The problem here with that is this VoiceSessionId will never be populated from when I was testing this. I would have further testing done to see what can be done with that issue

Reply:
I can't repro this but I'll take your word for it. Definitely doesn't work though when the bot loses connection, so I'll keep looking into this. I would like to know how you produced this though, so I can test it before pushing another commit.

Reply:
LOL, I think I actually solved this. I reverted all the session id functions back, and I tried finding a better place to get the voice session id, and it turns out that UserVoiceStateUpdated is called, it just wasn't already being caught with the current method so I didn't know about it. 😅 Anyway, this *should* work now, and will only be called *hopefully* when the client reconnects and the session id isn't already set. I actually like this way better.

Reply:
Also, I pushed the log message inside of the actual player / session id check because I didn't want it to spit out "VOICE_STATE_UPDATE (yada) Updated" and using Reconnected would've been incorrect if we weren't actually reconnecting! 😄

Reply:
Okay, does the new code still do this?